### ISSUE TITLE
Implement design on "playlists" "page" {1e}

### ISSUE THREAD

**Issue by felipegaucho** [Paratii-Video/paratii-player#5]
As in [here](https://github.com/Paratii-Video/paratii-design/blob/master/mvp-design/artboards/mvp-playlist.png)

This depends on #10

**Comment by jellegerbrandy** [comment]
duplicate of #10
